src/DecileStatistics.java

```diff
@@ -0,0 +1,40 @@
+import java.util.Arrays;
+import java.util.Scanner;
+
+public class DecileStatistics {
+    public static void main(String[] args) {
+        Scanner scanner = new Scanner(System.in);
+        int[] nums = new int[1000];
```

Comment:
what happens when I can put 5000 numers to calc deciles?

Reply:
ok but in requirements i did't see anything that I can calculate only for 1K numbers.  please change implementation to can read more numbers  (unlimited- but limited by available memory)

Reply:
Sorry, my bad. Already done.